### PR TITLE
[CR-26] upgrade colorpicker.

### PR DESCRIPTION
**Pull request by manachynskyi**
This PR is going to upgrade jquery_colorpicker module

Steps for review:
- [x] visit /admin/modules -> find Jquery Colorpicker
- [x] make sure jquery_colorpicker module updated https://i.imgur.com/0PONWCX.png
- [x] Edit homepage
- [x] change color of banner in header https://i.imgur.com/9jd5jLv.png
- [x] make sure alerts color are correct

### CODE REVIEW

**Comment by gundevel** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by gundevel** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by AndreyMaximov** [comment]
ok to test

**Comment by gundevel** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2584/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2584/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2584/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2584/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2584/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2584/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2584/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2584/

**Comment by gundevel** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2585/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2585/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2585/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2585/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2585/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2585/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2585/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2585/

**Comment by podarok** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2590/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2590/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2590/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2590/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2590/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2590/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2590/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2590/

**Comment by podarok** [comment]
image
doesn't work @manachynskyi 
Is this expected behavior?

**Comment by gundevel** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2591/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2591/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2591/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2591/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2591/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2591/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2591/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2591/

**Comment by podarok** [comment]
```
failed: [localhost] => {"changed": true, "cmd": "APP_ENV=dev drush -y si openy openy_theme_select.theme=openy_carnation openy_terms_of_use.agree_openy_terms=1 install_configure_form.enable_update_status_emails=NULL --db-url=mysql://root:root@127.0.0.1:/drupal2591 --account-name=admin --account-pass=open9622 --uri=http://openy.cibox.tools/build2591", "delta": "0:10:49.565194", "end": "2020-09-16 10:35:30.913118", "rc": 1, "start": "2020-09-16 10:24:41.347924", "warnings": []}
stderr: Starting Drupal installation. This takes a while. Consider using the        [ok]
--notify global option.
Drupal\Component\Plugin\Exception\PluginNotFoundException: The           [error]
"colorapi_color_field" plugin does not exist. Valid plugin IDs for
Drupal\Core\Field\FieldTypePluginManager are: address_country,
address, address_zone, datetime, daterange,
entity_reference_revisions, file_uri, file, geolocation, image, link,
metatag, openy_custom_hours, list_integer, list_string, list_float,
path, plugin:plugin_selector, plugin:entity_reference_selector,
plugin:field_widget, plugin:field_formatter, plugin:archiver,
plugin:action, plugin:menu_link, plugin:menu_local_action,
plugin:menu_local_task, plugin:menu_lcontextual_link,
plugin:display_variant, plugin:queue_worker, plugin:mail,
plugin:condition, plugin:element, plugin:block,
plugin:ckeditor_plugin, plugin:editor, plugin:filter,
plugin:image_effect, plugin:language_negotiation_method,
plugin:quickedit_editor, plugin:rest_resource, plugin:tour_tip,
plugin:views_access, plugin:views_area, plugin:views_argument,
plugin:views_argument_default, plugin:views_argument_validator,
plugin:views_cache, plugin:views_display_extender,
plugin:views_display, plugin:views_exposed_form, plugin:views_field,
plugin:views_filter, plugin:views_join, plugin:views_pager,
plugin:views_query, plugin:views_relationship, plugin:views_row,
plugin:views_sort, plugin:views_style, plugin:views_wizard,
plugin:paragraphs.paragraphs_behavior, redirect_source, telephone,
text_long, text_with_summary, text, video, video_embed_field, string,
uuid, timestamp, changed, language, string_long, email, map, float,
entity_reference, boolean, uri, integer, created, decimal, password
in
/var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace/build2591/openy-project/docroot/core/lib/Drupal/Component/Plugin/Discovery/DiscoveryTrait.php:53
Stack trace:
#0
/var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace/build2591/openy-project/docroot/core/lib/Drupal/Component/Plugin/Discovery/DiscoveryCachedTrait.php(25):
Drupal\Core\Plugin\DefaultPluginManager->doGetDefinition(Array,
'colorapi_color_...', true)
```

**Comment by gundevel** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2592/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2592/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2592/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2592/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2592/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2592/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2592/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2592/

**Comment by gundevel** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2595/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2595/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2595/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2595/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2595/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2595/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2595/blogWCAG2AAhtmlcs.txt

**Comment by AndreyMaximov** [comment]
retest this please

**Comment by gundevel** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2595/

**Comment by gundevel** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2596/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2596/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2596/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2596/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2596/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2596/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2596/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2596/

**Comment by gundevel** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2597/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2597/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2597/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2597/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2597/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2597/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2597/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2597/

**Comment by podarok** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2598/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2598/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2598/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2598/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2598/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2598/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2598/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2598/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2598/

File: modules/openy_features/openy_demo_content/modules/openy_demo_tcolor/config/install/migrate_plus.migration.openy_demo_taxonomy_term_color.yml [hunk 1]
```diff
@@ -14,71 +14,71 @@ source:
...
     -
       id: 4
       name: Light blue
-      color: '00AEEF'
+      color: '#00AEEF'
     -
       id: 5
       name: Blue
-      color: '0089D0'
+      color: '#0089D0'
```
Comment (podarok) [inline modules/openy_features/openy_demo_content/modules/openy_demo_tcolor/config/install/migrate_plus.migration.openy_demo_taxonomy_term_color.yml:33]: How we support upgrade path here?
Will current color vocabularies work?
Should we create hook_update_N for those?

Comment (podarok) [reply modules/openy_features/openy_demo_content/modules/openy_demo_tcolor/config/install/migrate_plus.migration.openy_demo_taxonomy_term_color.yml:33]: I see it works 
image

**Pull request merged**